examples/jquery-bootstrap/js/app.js

```diff
@@ -19,9 +19,13 @@ var BootstrapModal = React.createClass({
   componentDidMount: function() {
     // When the component is added, turn it into a modal
     $(this.refs.root).modal({backdrop: 'static', keyboard: false, show: false});
+
+    // Bootstrap's modal class exposes a few events for hooking into modal
+    // functionality. Lets hook into one of them:
+    $(this.refs.root).on('hidden.bs.modal', this.handleHidden);
   },
   componentWillUnmount: function() {
-    $(this.refs.root).off('hidden', this.handleHidden);
+    $(this.refs.root).off('hidden.bs.modal', this.handleHidden);
```

Comment:
Also, it may not have been working because there was an action listener in place for either event ie.

```javascript
$(this.refs.root).on('hidden.bs.modal', this.handleHidden);
// or
$(this.refs.root).on('hidden', this.handleHidden);
```
I added that listener on line 25